### ISSUE TITLE
One shot translation and translation history

### ISSUE THREAD

**Issue by realyasster** [Danily07/Translumo#21]
Hi again :D 

Translation doesn't always need to be consecutive. Can you provide support for snapshot translation? Automatic translation will be performed when an area is selected. You can add new shortcut for this. In addition, translation histories should be saved. You can add an option to the menu for this.

Thanks.

**Comment by Danily07** [comment]
Snapshot (one-time translation) is alrady available (shift+F by default). Or I just misunderstood your feature description?
As for history saving, I will consider adding that option, thanks.

**Comment by realyasster** [comment]
> Snapshot (one-time translation) is alrady available (shift+F by default). Or I just misunderstood your feature description?

After selecting the field, we need to start the translation. Am I wrong? Once the translation is started, it continuously translates the selected area. In the case I mentioned, it automatically performs a one-time translation directly after the area is selected.

**Comment by realyasster** [comment]
> Snapshot (one-time translation) is alrady available (shift+F by default). Or I just misunderstood your feature description?

After selecting the field, we need to start the translation. Am I wrong? Once the translation is started, it continuously translates the selected area. In the case I mentioned, it automatically performs a one-time translation directly after the area is selected.

**Comment by Danily07** [comment]
> After selecting the field, we need to start the translation. Am I wrong?

No, in this mode translation will be displayed automatically once after area selection and it doesn't affect real-time mode.

**Comment by realyasster** [comment]
> No, in this mode translation will be displayed automatically once after area selection and it doesn't affect real-time mode.

Oh that's my bad. I miss that.

**Comment by EerFree** [comment]
> Snapshot (one-time translation) is alrady available (shift+F by default). Or I just misunderstood your feature description? As for history saving, I will consider adding that option, thanks.

add hot keys options for more zones, please, like they do in the Maverick OCR Helper

**Comment by Kaminekochan** [comment]
> Snapshot (one-time translation) is alrady available (shift+F by default). Or I just misunderstood your feature description? As for history saving, I will consider adding that option, thanks.

it would be useful to save the selected area, and do the translation not automatically, but at the press of a button, that way we don't load the translators with requests and I don't recall the block from the translators. Dialogue lines are often slow, having them do a translation of a saved area only at the press of a key and not continuously is more useful. This will also make it possible to assign multiple zones to multiple keys.
